# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/TransceiverImpl.h
# Commit: e0a2e54e4de8
# Full Hash: e0a2e54e4de80813829cbcf9e4a5e4ff1285d451
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 9
# Description:
#   Bug 1654112 - Move WebRtcCallWrapper to dedicated files. r=ng
#   
#   This patch also renames it WebrtcCallWrapper, to better match neighboring
#   classes starting on "Webrtc".
#   
# ==============================================================================

diff -r 1965757ee924 -r e0a2e54e4de8 dom/media/webrtc/jsapi/TransceiverImpl.h
--- a/dom/media/webrtc/jsapi/TransceiverImpl.h	Thu Sep 02 13:22:03 2021 -0500
+++ b/dom/media/webrtc/jsapi/TransceiverImpl.h	Thu Apr 08 11:31:43 2021 +0200
@@ -29,7 +29,7 @@
 class MediaPipeline;
 class MediaPipelineFilter;
 class MediaTransportHandler;
-class WebRtcCallWrapper;
+class WebrtcCallWrapper;
 class JsepTrackNegotiatedDetails;
 
 namespace dom {
@@ -62,7 +62,7 @@
                   nsISerialEventTarget* aMainThread,
                   nsISerialEventTarget* aStsThread,
                   dom::MediaStreamTrack* aSendTrack,
-                  WebRtcCallWrapper* aCallWrapper);
+                  WebrtcCallWrapper* aCallWrapper);
 
   bool IsValid() const { return !!mConduit; }
 
@@ -164,7 +164,7 @@
   nsCOMPtr<nsISerialEventTarget> mStsThread;
   RefPtr<dom::MediaStreamTrack> mSendTrack;
   // state for webrtc.org that is shared between all transceivers
-  RefPtr<WebRtcCallWrapper> mCallWrapper;
+  RefPtr<WebrtcCallWrapper> mCallWrapper;
   RefPtr<MediaSessionConduit> mConduit;
   RefPtr<MediaPipelineTransmit> mTransmitPipeline;
   // The spec says both RTCRtpReceiver and RTCRtpSender have a slot for